### PR TITLE
Beautifing location card

### PR DESCRIPTION
**Pull request by uvluni**
Some custom CSS added to the Grid and Card components.

### CODE REVIEW

File: src/components/LocationCard/LocationCard.js [hunk 1]
```diff
@@ -1,25 +1,35 @@
+import 'antd/lib/tooltip/style/css';
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Tooltip } from 'antd';
 
-import { SelectEffect } from '../LocationCard/LocationCard.styled';
+import {
+  SelectEffect,
+  P,
```
Comment (Alaev) [inline src/components/LocationCard/LocationCard.js]: P is not a good name.
Try to find a more specific name here. title, text, paragraph etc..

File: src/components/LocationCard/LocationCard.js [hunk 2]
```diff
@@ -1,25 +1,35 @@
...
+  BtnsWrapper,
+  CallToActionBtn,
+  MapBtn
```
Comment (Alaev) [inline src/components/LocationCard/LocationCard.js]: I would change it to CallToActionMap

Comment (uvluni) [reply src/components/LocationCard/LocationCard.js]: The CallToActionBtn is the button "Call now" and "Book a seat". The map is the other one.

File: src/components/LocationCard/LocationCard.js (hunk 1 above)
Comment (uvluni) [reply src/components/LocationCard/LocationCard.js]: Done

**Comment by Alaev** [comment]
@uvluni push changed to brach for review

**Comment by uvluni** [comment]
Pushed the branch

**Comment by Alaev** [comment]
Merged

**Pull request merged**